.changeset/nice-news-peel.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-css': minor
+'@microsoft/atlas-site': minor
+---
+
+Adding table modifiers; updating site with relevant documenation
```

Comment:
"documentation."?

Reply:
Updated

Reply:
still seeing "documenation" instead of "documentation"

Reply:
Oh, my fault, I thought you were pointing the missed comma! Updated now